<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConditionWidget</class>
 <widget class="QWidget" name="ConditionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>399</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>398</width>
    <height>399</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="m_pOperatorWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QRadioButton" name="m_pEqual">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>==</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_pLessEqual">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;=</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_pLess">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_pGreater">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_pGreaterEqual">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;=</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_pNotEqual">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>!=</string>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="m_pType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>381</width>
     <height>351</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>381</width>
     <height>351</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="m_pTypeVariables_2">
    <attribute name="title">
     <string>Variables</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>351</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Destination Variable:</string>
     </property>
     <widget class="QTabWidget" name="m_pDestTab">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="m_pDestGlobal">
       <attribute name="title">
        <string>Global Variable</string>
       </attribute>
       <widget class="QComboBox" name="m_pGlobalVarBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>311</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="m_pDestLocalVariable">
       <attribute name="title">
        <string>Local Variable</string>
       </attribute>
       <widget class="QComboBox" name="m_pLocalVarBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>311</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="m_pVariableTypeWidget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
     <widget class="QRadioButton" name="m_pInteger">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>0</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>integer</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="m_pString">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>0</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>string</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="m_pBool">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>bool</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="m_pFloat">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>0</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>float</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>351</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Source:</string>
     </property>
     <widget class="QTabWidget" name="m_pSrcTab">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>331</width>
        <height>151</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="m_pSrcValue">
       <attribute name="title">
        <string>Value</string>
       </attribute>
       <widget class="QLabel" name="label_36">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>bool value:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_37">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>integer value:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_38">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>float value:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_39">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>string value:</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="m_pValueFloat">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>70</y>
          <width>191</width>
          <height>22</height>
         </rect>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>-9999999.999900000169873</double>
        </property>
        <property name="maximum">
         <double>9999999.999900000169873</double>
        </property>
       </widget>
       <widget class="QSpinBox" name="m_pValueInteger">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>40</y>
          <width>191</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-2147483647</number>
        </property>
        <property name="maximum">
         <number>2147483647</number>
        </property>
       </widget>
       <widget class="QLineEdit" name="m_pValueString">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>100</y>
          <width>191</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="m_pValueBool">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>10</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::InsertAlphabetically</enum>
        </property>
        <item>
         <property name="text">
          <string>false</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>true</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QWidget" name="m_pSrcGlobalVariable">
       <attribute name="title">
        <string>Global Variable</string>
       </attribute>
       <widget class="QLabel" name="label_40">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>string value:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_41">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>integer value:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_42">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>bool value:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_43">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>float value:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="m_pSrcGlobalBoolBox">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>10</y>
          <width>191</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="m_pSrcGlobalIntegerBox">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>40</y>
          <width>191</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="m_pSrcGlobalFloatBox">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>70</y>
          <width>191</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="m_pSrcGlobalStringBox">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>100</y>
          <width>191</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="m_pSrcLocalVariable">
       <attribute name="title">
        <string>Local Variable</string>
       </attribute>
       <widget class="QLabel" name="label_44">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>integer value:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="m_pSrcLocalBoolBox">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>10</y>
          <width>191</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="m_pSrcLocalFloatBox">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>70</y>
          <width>191</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_45">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>bool value:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="m_pSrcLocalIntegerBox">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>40</y>
          <width>191</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_46">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>string value:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_47">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>float value:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="m_pSrcLocalStringBox">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>100</y>
          <width>191</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
